src/js/components/FormField/FormField.js

```diff
@@ -502,7 +509,20 @@ const FormField = forwardRef(
         }}
         onBlur={(event) => {
           setFocus(false);
-          if (contextOnBlur) contextOnBlur(event);
+
+          // if input has a drop and focus is within drop
+          // prevent onBlur validation from running until
+          // focus is no longer within the drop or input
+          const dropPortalId = withinDropPortal(event.relatedTarget);
```

Comment:
What about pushing more of this logic into the utility function? For example:
```
if (contextOnBlur && !formFieldRef.current.contains(event.relatedTarget) && !withinPortalContext(event.relatedTarget, portalContext)) {
  contextOnBlur(event);
}
```

Reply:
done